internal/xds/bootstrap/tlscreds/bundle.go

```diff
@@ -136,3 +149,38 @@ func (c *reloadingCreds) OverrideServerName(string) error {
 func (c *reloadingCreds) ServerHandshake(net.Conn) (net.Conn, credentials.AuthInfo, error) {
 	return nil, nil, errors.New("server handshake is not supported by xDS client TLS credentials")
 }
+
+func buildSPIFFEVerifyFunc(spiffeBundleMap spiffe.BundleMap, peerVerifiedChains [][]*x509.Certificate) func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
```

Comment:
Can we add unit tests for this function?

Reply:
I think given that it is unexpected we unit test via exported functions - and, since this builds and returns a function it's a little stranger to unit test I think.

Reply:
Fine by me, so long as we are indeed testing the exported function that wraps this.